### PR TITLE
Remove MMLU step from phased training 

### PR DESCRIPTION
**Pull request by MichaelClifford**
Addresses #56 

This PR primarily comments out all the MMLU eval code to remove it from the pipeline. Once we confirm none of it is needed for the final eval pieces we can remove it in a follow up PR. 

This PR also adds a new function to `pytorchjob_manifest_op` that will promote the most recent model checkpoint from the first training phase as the starting model for the second training phase.

### CODE REVIEW

File: standalone.tpl [hunk 1]
```diff
@@ -282,7 +282,7 @@ def cli():
 @click.option(
     "--eval-type",
     help="Type of evaluation to run",
-    type=click.Choice(["mmlu", "mt-bench"]),
+    type=click.Choice(["mt-bench"]),
```
Comment (leseb) [inline standalone.tpl:285]: We could get rid of this flag entirely IMO. If we don't please add `hidden=True` to not show it in the help.

File: training/components.py [hunk 2]
```diff
@@ -85,17 +85,29 @@ def pytorchjob_manifest_op(
...
     nproc_per_node: int = 3,
     nnodes: int = 2,
 ) -> NamedTuple("outputs", manifest=str, name=str):
     import inspect
+    import os
+
+    def list_phase1_final_model():
+        model_dir = "/output/model/hf_format"
+        models = os.listdir(model_dir)
+        newest_idx = max([(int(j.split("_")[-1]), i) for i, j in enumerate(models)])[-1]
```
Comment (leseb) [inline training/components.py:99]: hum, how about we use `mtime` to sort?

```suggestion
        newest_idx = max((os.path.getmtime(model), i) for i, model in enumerate(models))[-1]
```

**Review by leseb — COMMENTED** [review]
just nits, otherwise LGTM!

File: training/components.py (hunk 2 above)
Comment (MichaelClifford) [reply training/components.py:99]: Much more reliable approach :) Thanks!

**Review by leseb — APPROVED** [review]

**Pull request merged**